usagov/vote-gov-drupal
# VOTE-1591 Add Inline alert block type

## Jira ticket (required)
[VOTE-1591](https://bixal-projects.atlassian.net/browse/VOTE-1591)

## Description (optional)
Adds a new Inline Alert block type to the CMS

## Deployment and testing (required, if applicable)
### Post-deploy steps
1. `lando retune`

### Testing steps
1. Visit http://vote-gov.lndo.site/admin/content/site-alerts
2. Click on "Add inline Alert" and create a new alert
3. Confirm it is in the listing on http://vote-gov.lndo.site/admin/content/site-alerts when you filter by "Inline Alert"
